build.gradle.kts

```diff
@@ -76,7 +79,7 @@ val generateSvelteLexer = task<GenerateLexerTask>("generateSvelteLexer") {
 }
 
 tasks {
-    val javaVersion = "11"
+    val javaVersion = "17"
```

Comment:
Please don't change more than necessary, let's stay on Java 11 & Kotlin 1.5 for maximum compatibility